tsl/src/bgw_policy/job.c

```diff
@@ -380,6 +381,13 @@ policy_refresh_cagg_execute(int32 job_id, Jsonb *config)
 
 	policy_refresh_cagg_read_and_validate_config(config, &policy_data);
 
+	/* See if there is a move hypertable invalidations policy defined */
+	List *jobs =
+		ts_bgw_job_find_by_proc_and_hypertable_id(POLICY_PROCESS_HYPER_INVAL_PROC_NAME,
+												  FUNCTIONS_SCHEMA_NAME,
+												  policy_data.cagg->data.raw_hypertable_id);
+	bool has_move_policy = list_length(jobs) > 0;
```

Comment:
We might want to add a GUC to allow this to be configured so that the default behavior is to always process the hypertable invalidations but it can be enabled.

Reply:
Maybe we can use the existing `force` option

Reply:
Yes, this is what the code currently does, but that means you have to pass it explicitly with each call.

I am thinking that you might want to set it up in one of three different ways:

1. Always do an invalidations move when running a refresh
2. Never do an invalidation move when running a refresh
3. Do an invalidation move when there is no move policy defined.